blocks/subscriptions-block/components/GridList/index.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from '@arc-fusion/prop-types';
+
+import './styles.scss';
+
+const GridList = ({ children }) => {
+  const additionalClass = children.length > 3 ? `xpmedia-subscription-grid-list--${children.length}` : '';
```

Comment:
hmm haven't seen this kind of naming with grid before. seems like it'd work well though